x-pack/plugins/observability_solution/slo/common/constants.ts

```diff
@@ -87,6 +86,9 @@ export const DEFAULT_SLO_GROUPS_PAGE_SIZE = 25;
 export const getSLOSummaryTransformId = (sloId: string, sloRevision: number) =>
   `slo-summary-${sloId}-${sloRevision}`;
 
+export const getSLOPipelineId = (sloId: string, sloRevision: number) =>
+  `.slo-observability.sli.pipeline-${sloId}-${sloRevision}`;
```

Comment:
does 64 chars constraint applies here?

Reply:
Apparently pipeline id does not have a 64 char limit like the transform id